docs/pages/_document.js

```diff
@@ -39,7 +39,7 @@ export default class MyDocument extends Document {
     const { canonicalAs, userLanguage } = this.props;
 
     return (
-      <Html lang={userLanguage}>
+      <Html lang={userLanguage} style={{ fontSize: '16px' }}>
```

Comment:
This breaks the accessibility (configurable font size)

```suggestion
      <Html lang={userLanguage}>
```